src/Microsoft.CodeAnalysis.Testing/README.md

```diff
@@ -46,15 +46,15 @@ To reference prerelease packages, add a **NuGet.Config** file to your solution d
 ### xUnit.net
 
 * C#
-  * Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit
-  * Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit
-  * Microsoft.CodeAnalysis.CSharp.CodeRefactoring.Testing.XUnit
-  * Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing.XUnit
+  * Microsoft.CodeAnalysis.CSharp.Analyzer.Testing
+  * Microsoft.CodeAnalysis.CSharp.CodeFix.Testing
+  * Microsoft.CodeAnalysis.CSharp.CodeRefactoring.Testing
+  * Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing
 * Visual Basic
-  * Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing.XUnit
-  * Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.XUnit
-  * Microsoft.CodeAnalysis.VisualBasic.CodeRefactoring.Testing.XUnit
-  * Microsoft.CodeAnalysis.VisualBasic.SourceGenerators.Testing.XUnit
+  * Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing
+  * Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing
+  * Microsoft.CodeAnalysis.VisualBasic.CodeRefactoring.Testing
+  * Microsoft.CodeAnalysis.VisualBasic.SourceGenerators.Testing
```

Comment:
This very much worries me here that we're making a source breaking change more or less for folks who consumed this. It seems a much smaller change would just be to remove the verifier implementation that is causing trouble here, and leave the packages and core scaffolding in place. This way:

1. People who have to migrate their packages forward won't have to change a bunch of package references/usings.
2. If we still have xunit specific value we do want to provide (say, xunit does offer a way to have richer exceptions), we have a natural process for restoring that without asking everybody to change their package references/usings _again_.

Reply:
We published a public 1.2 release with these types marked obsolete, finally giving us a path forward to remove them. 🎉